### DISCUSSION TITLE
It is ready for testing?

### DISCUSSION THREAD

**Discussion by Frederick-G764** [ThemeParkWorld/OpenTPW#10] [Q&A]
Hi,

That's look like a stunning project. Is this already stable for testing? And what that's with the "Engine @ 8ce2d29".

Best regards

**Comment by xezno** [comment]
Not at the moment. There's still a lot of work that needs doing, especially with the various file formats that the game uses.

**Comment by Frederick-G764** [comment]
thanks for the update. Looking forward to a release 👍

**Comment by weegie247** [comment]
Hello! Definitely keen to see this come back in some shape or form! Is this still active?

**Comment by xezno** [comment]
Hey, sorry for the late reply! It's not inactive but it's not really active right now either. There isn't a ton of stuff I can do until I get around to reversing the mesh/texture file formats, and I'm a little busy with some personal stuff at the moment, so there won't be much here until that changes (and I'm not sure when that will be).

**Comment by weegie247** [comment]
> Hey, sorry for the late reply! It's not inactive but it's not really active right now either. There isn't a ton of stuff I can do until I get around to reversing the mesh/texture file formats, and I'm a little busy with some personal stuff at the moment, so there won't be much here until that changes (and I'm not sure when that will be).

Hey! No worries at all, hope you’re well and healthy. Will be interesting to see how you do when you come back to it. Definitely a memorable game.

**Comment by ViolentKitty3** [comment]
Hi, I created an account just for this thread. I was wondering of this project was still active? Would love for someone just make this a playable game for everyone again.

**Comment by xezno** [comment]
Yeah I'm actively working on it right now

**Comment by weegie247** [comment]
Hey Alex,

Brilliant news. Best of luck with it! I’ve just had the email notification
so not seen the GitHub recently but I’m excited to see what you get up to.

I’d offer to help but I could only realistically offer my video editing
skills if you need videos made etc

Again, enjoy and all the best.

On Fri, 11 Mar 2022 at 12:11 am, Alex ***@***.***> wrote:

> Yeah I'm actively working on it right now
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ThemeParkWorld/OpenTPW/issues/1#issuecomment-1064641805>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATJ6QNRPYP6PCSTMDBEBHCDU7KFSPANCNFSM4PGAP5AQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Kind Regards,
Ben Glasgow

www.BenGlasgow.co.uk

**Comment by xezno** [comment]
I'm going to move this issue to a discussion because I think it's more fitting than having an issue like this open - issues are really just for bug reports. I think eventually I'll also start a second discussion thread with some screenshots and progress (once there's some decent progress to show off)..

**Comment by KaylaD23** [comment]
hope this stays active cuz its really a pain to get the original game to work on windows 11 especially on intel arc as it has problems i fixed said problems a while ago now but i forgot what i did to fix it soo right now im in a state of if i loose the game like say a hard drive dies or something theres a good chance i would of forgotten what i did to fix some issues if i had to re install the game onto my pc again
